circle.yml

```diff
@@ -4,7 +4,6 @@ machine:
 
 dependencies:
   post:
-    - bundle exec rake lint:install
```

Comment:
Same questions as in https://github.com/SwiftGen/SwiftGen/pull/401#discussion_r177748654

Reply:
Same answer :trollface: